src/Text/Pandoc/Options.hs

```diff
@@ -267,39 +285,40 @@ data WriterOptions = WriterOptions
-                      , writerHTMLMathMethod   = PlainMath
-                      , writerNumberSections   = False
-                      , writerNumberOffset     = [0,0,0,0,0,0]
-                      , writerSectionDivs      = False
-                      , writerExtensions       = emptyExtensions
-                      , writerReferenceLinks   = False
-                      , writerDpi              = 96
-                      , writerWrapText         = WrapAuto
-                      , writerColumns          = 72
-                      , writerEmailObfuscation = NoObfuscation
-                      , writerIdentifierPrefix = ""
-                      , writerCiteMethod       = Citeproc
-                      , writerHtmlQTags        = False
-                      , writerSlideLevel       = Nothing
-                      , writerTopLevelDivision = TopLevelDefault
-                      , writerListings         = False
-                      , writerHighlightStyle   = Just pygments
-                      , writerSetextHeaders    = True
-                      , writerEpubSubdirectory = "EPUB"
-                      , writerEpubMetadata     = Nothing
-                      , writerEpubFonts        = []
-                      , writerEpubChapterLevel = 1
-                      , writerTOCDepth         = 3
-                      , writerReferenceDoc     = Nothing
-                      , writerReferenceLocation = EndOfDocument
-                      , writerSyntaxMap        = defaultSyntaxMap
-                      , writerPreferAscii      = False
-                      }
+  def =
+    WriterOptions { writerTemplate              = Nothing
+                  , writerVariables             = mempty
+                  , writerTabStop               = 4
+                  , writerTableOfContents       = False
+                  , writerIncremental           = False
+                  , writerHTMLMathMethod        = PlainMath
+                  , writerNumberSections        = False
+                  , writerNumberOffset          = [0,0,0,0,0,0]
+                  , writerSectionDivs           = False
+                  , writerExtensions            = emptyExtensions
+                  , writerReferenceLinks        = False
+                  , writerDpi                   = 96
+                  , writerWrapText              = WrapAuto
+                  , writerColumns               = 72
+                  , writerEmailObfuscation      = NoObfuscation
+                  , writerIdentifierPrefix      = ""
+                  , writerCiteMethod            = Citeproc
+                  , writerHtmlQTags             = False
+                  , writerSlideLevel            = Nothing
+                  , writerTopLevelDivision      = TopLevelDefault
+                  , writerListings              = False
+                  , writerHighlightStyle        = Just pygments
+                  , writerMarkdownHeadingFormat = HeadingFormatAtx
```

Comment:
Shouldn't the default be Setext?  (In principle we could have a different default than with command line pandoc, but I think it makes sense to keep the same default.)